packages/@magic-ext/solana/package.json

```diff
@@ -20,7 +20,8 @@
   "react-native": "./dist/react-native/index.native.js",
   "exports": {
     "import": "./dist/es/index.mjs",
-    "require": "./dist/cjs/index.js"
+    "require": "./dist/cjs/index.js",
+    "./package.json": "./package.json"
```

Comment:
I see a linter warning coming from VSCode's `package.json` schema-checker. Not sure if this is actually an issue in practice. Do you have any context @bmeeder22?

The warning disappears when conditional `"exports"` are not in use. I'm not sure if this is a limitation of the spec, or of the schema reference.

<img width="426" alt="Screen Shot 2022-01-25 at 7 55 52 PM" src="https://user-images.githubusercontent.com/22753882/151096493-4aa4f337-ffca-4c8f-ba30-67afb9db548d.png">

Reply:
I've seen it as best practices here: https://github.com/uuidjs/uuid/issues/444#issuecomment-630424195

Reply:
The place that I am seeing warnings is while attempting to use this package in `react-native`:

yarn run v1.22.17
```
$ react-native start
warn Package @magic-ext/solana has been ignored because it contains invalid configuration. Reason: Package subpath './package.json' is not defined by "exports" in /Users/.../mobile/node_modules/@magic-ext/solana/package.json
```

This appears to be unique to react native >= Node V14 as per this issue: https://github.com/react-native-community/cli/issues/1168

Reply:
Because the warning is in the VS Code linter and not eslint. My guess is that your seeing that error only as part of the VS Code built in linter which might not have the most updated spec. 

It appears that this only matters when there are defined exports. In packages without explicitly defined exports everything works just fine.

Reply:
Another good thing to note that the warning appears to just be a warning initializing the extension is working just fine in the RN application. Hopefully that doesn't change in some later version.